pkg/publish/servicelb.go

```diff
@@ -33,6 +36,18 @@ func ServiceLoadBalancer(svc *v1.ServiceInstance) (result []kclient.Object, _ er
 		return nil, nil
 	}
 
+	cfg, err := config.Get(req.Ctx, req.Client)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, entry := range typed.Sorted(cfg.ServiceLBAnnotations) {
+		if svc.Spec.Annotations == nil {
+			svc.Spec.Annotations = map[string]string{}
+		}
```

Comment:
Can this check be pulled out of the for-loop?

Reply:
Yeah totally, I was a little nervous about setting it to empty when it was nil but thinking on it more its likely a no-op.